Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
i did not change pom.xml in text manner.... I added from the dependencies tab
so i have nothing to do with mis spelling.... also can you tell me what mis-spelling you are mentioning about?